info.csv

```diff
@@ -18,4 +18,5 @@ Himanshu Singh,          17SCSE101209,           1713101199,            None,
 PRIYANSHU TRIVEDI,       17SCSE105051,           1713105051,            NONE,                            2nd,         1-cloud,  C C++ PYTHON,                 competitive coding,        NA,                  trivedipriyanshu972@gmail.com,    https://github.com/PRIYANSHU-AMBITION,   https://www.linkedin.com/in/priyanshu-trivedi-54020b189/,  NA
 Ankit Singh,             17SCSE114059,           1713114009,            Web Development,                 2,           1,        "python and HTML CSS",        Web development and AI,   NA,                   ankitsingh4301@gmail.com,         https://github.com/1AnkitSingh02,       https://www.linkedin.com/in/ankit-singh-1b968a170,          NA
 ketan singh,             17SCSE101558,           1713101536,            Web development,                 2,           2,         NA,                          web development,            NA,                  ketansingh343@gmail.com,         https://github.com/KeTan05,              https://www.linkedin.com/in/ketan-singh-a6093816b/,        NA
-Rishabh Arora,           18SCSE1010692,          18021011915,           Web Development,                 1,           2,         HTML CSS JavaScript,         Web Development,          Previous School Website, rishabharora.ns@gmail.com,     https://github.com/RishabhAroraa,       https://www.linkedin.com/in/rishabh-arora-b78333156/,       NA
\ No newline at end of file
+Rishabh Arora,           18SCSE1010692,          18021011915,           Web Development,                 1,           2,         HTML CSS JavaScript,         Web Development,          Previous School Website, rishabharora.ns@gmail.com,     https://github.com/RishabhAroraa,       https://www.linkedin.com/in/rishabh-arora-b78333156/,       NA
+Sakshi Gupta,             17SCSE1012000,         1713105032,            Android development,         2,           13,        Core java,Python(basic),                  Competitive Coding,       GUI based app.,          sakshigupta06122000@gmail.com,  https://github.com/sakshigupta06,      https://www.linkedin.com/in/sakshigupta,                    NA
```

Comment:
Wrap multiple values in `" "`
`Core java,Python(basic) ` should be `"Core java,Python(basic), "`